docker-compose.yaml

```diff
@@ -19,7 +19,7 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
-    image: openfga/openfga:latest
+    image: openfga/openfga:v1.1.1
```

Comment:
🤔 why? latest already includes 1.1.1. The digests are the same: https://hub.docker.com/r/openfga/openfga/tags

Reply:
I didn't intend to check in the docker-compose.yaml file.  Will revert that file.